main/newscorm/lp_controller.php

```diff
@@ -380,7 +380,7 @@ function(reponse){
         if (isset($_REQUEST['action']) && $_REQUEST['action'] == 'view' && !isset($_REQUEST['exeId'])) {
             $_REQUEST['action'] = 'build';
         }
-        //$_SESSION['studentview'] = null;
```

Comment:
That value is only modified in local.inc.php.

Maybe adding in the link "&isStudentView=false" will fix the problem::
<pre>
main/newscorm/lp_controller.php?cidReq=TEST&id_session=0&gidReq=0&origin=&action=add_item&type=step&lp_id=1&isStudentView=false
</pre>

instead of 
<pre>
main/newscorm/lp_controller.php?cidReq=TEST&id_session=0&gidReq=0&origin=&action=add_item&type=step&lp_id=1
</pre>

When clicking in the edit button as a teacher